lua/ulx/modules/sh/yeet.lua

```diff
@@ -0,0 +1,38 @@
+local CATEGORY_NAME = "Fun"
+
+local MIN_STRENGTH = 6000	
+local MAX_STRENGTH = 500000	
+local DEFAULT_STRENGTH = 6000 
+
+function yeetPlayer( caller, targets, strength )
+	
+	for _, v in pairs( targets ) do 
+		
+	if not v:IsPlayer() then return end
+	if not v:Alive() then return end
```

Comment:
I think you'd just want to skip dead/not-players here instead of breaking out of the entire function. `return` will halt the function entirely, not just go to the next iteration.

In most languages, you're given a `continue` or `next` which means "skip this iteration and go to the next". Sort of like a `return` for a loop.

Lua, unfortunately, doesn't provide this. Garry does, but his implementation is very poor and is discouraged by the wiki ([ref](https://wiki.facepunch.com/gmod/Specific_Operators)), so we don't use it.

Instead, just nesting the rest of the loop inside of a conditional is advisable. Something like this:

```lua
for _, ply in pairs( targets ) do
    local canYeet = IsValid( ply ) and ply:IsPlayer() and ply:IsAlive()

    if canYeet then
        if ply:IsFrozen() then ply:Freeze( false ) end

        doYeet( ply )
    end
end
```